mitosheet/mitosheet/user/utils.py

```diff
@@ -110,24 +112,11 @@ def should_upgrade_mitosheet() -> bool:
     NOTE: if the user clicks the upgrade button in the app, then we change the upgraded 
     date to this date, so that the user doesn't get a bunch of annoying popups. This just
     pushes back when they are annoyed to upgrade!
+
+    NOTE: THIS FUNCTION IS NOT IN USE ANY MORE. WE NO LONGER TELL USERS TO UPGRADE
+    AS THERE IS NO REAL REASON FOR THEM TO DO SO, AND IT IS ANNOYING.
```

Comment:
If we're getting rid of this, then let's delete the whole function so its not hanging around our codebase. Want to do this in a separate PR?